ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -100,25 +100,28 @@
             label="Email Address"
             v-model="submittingParty.emailAddress"
             :rules="emailRules"
+            validate-on-blur
           />
 
           <!-- Phone Number -->
           <label class="generic-label" for="submitting-party-phone">Phone Number</label>
           <v-row no-gutters>
             <v-col>
               <v-text-field
+                v-mask="'(###) ###-####'"
```

Comment:
What does v-mask directive do exactly?
Is it a means of auto formatting?

Reply:
It provides the ability to add a mask to a field to limit imput and provide formatting.  Masks are a great way of limiting keyboard input and limiting field sizes. In our case when the user types a phone number they will see the number....

Tracy provided a good explanation in the ticket of why masks are used..